src/basilisp/core.lpy

```diff
@@ -4831,7 +4831,7 @@
   (->> (group-by first methods)
        (reduce (fn [m [method-name arities]]
                  (->> (map rest arities)
-                      (apply list `fn method-name)
```

Comment:
If the extended method function uses the dispatch method name, then the compiler will add the _local_ function name to the symbol table for that name and users will be unable to call the generic Protocol dispatch method.